### ISSUE TITLE
cli.js use CRLF instead of LF

### ISSUE THREAD

**Issue by AsakuraMizu** [koishijs/koishi#133]
**Describe the bug**

`cli.js`, which is expected to use LF line break type, uses CRLF so that it doesn't work.

**To Reproduce**

```shell
> yarn add koishi
> yarn koishi
yarn run v1.22.10
$ /home/asakuram/projects/SiGNAL/node_modules/.bin/koishi
/usr/bin/env: 'node\r': No such file or directory
error Command failed with exit code 127.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Expected behavior**

```shell
❯ yarn koishi
yarn run v1.22.10
$ /home/asakuram/projects/SiGNAL/node_modules/.bin/koishi
koishi/2.4.3

Usage:
  $ koishi <command> [options]

Commands:
  init [file]  initialize a koishi configuration file
  run [file]   start a koishi bot

For more info, run any command with the `--help` flag:
  $ koishi init --help
  $ koishi run --help

Options:
  -h, --help     Display this message 
  -v, --version  Display version number 
Done in 0.32s.
```

**Screenshots**

![Screenshot_20201231_202215](https://user-images.githubusercontent.com/17076564/103410348-d4fd7700-4ba5-11eb-9775-69ba95a6c06e.png)

**Versions**

- OS: Arch Linux
- Node Version: 14.15.1
- Koishi Version: 2.4.3

**Comment by Shigma** [comment]
@AsakuraMizu I have just published v2.5.0. Can you test on the new version?

**Comment by AsakuraMizu** [comment]
ok
